docs/src/_includes/layouts/docs.webc

```diff
@@ -171,10 +171,13 @@
         </div>
 
         <div class="docs-layout__center-wrapper | grid-area--page-sidebar">
-          <nav class="docs-layout__center-sidebar">
+          <div class="docs-layout__center-sidebar">
```

Comment:
Cela peut rester un `nav` sémantique et être assigné à un composant avec les bons attributs (`webc:is="nom-du-composant"`)